logbook-json/src/main/java/org/zalando/logbook/json/JsonHttpLogFormatter.java

```diff
@@ -62,11 +62,14 @@ public JsonHttpLogFormatter(final ObjectMapper mapper) {
     public Optional<Object> prepareBody(final HttpMessage message) throws IOException {
         final String contentType = message.getContentType();
         final String body = message.getBodyAsString();
-
+        if(body.isEmpty()) {
+            return Optional.of("");
```

Comment:
Now we have three return statements in here and all of them use `Optional.of(..)`. That means we never return an empty Optional now.

The original idea was to return empty for an empty/absent body. The contract of the JSON log output until also was that a request/response without body will produce an absent `body` field in the log message. I'd like to preserve that.

Reply:
Ok, I'd like that too, just make a minimal fix first. So then HttpMessage can return null in getBodyAsString() or what?

Reply:
I believe what we should do is replace line 66 with `return Optional.empty()`.